client/src/components/app.jsx

```diff
@@ -72,10 +91,10 @@ class App extends React.Component {
         <PhotoBody>
           <div className="photo-gallery-body">
             <div>
-              <Header Photos={this.state.photos} />
+              <Header Photos={this.state.photos} handleHeaderClick={this.handleHeaderClick} />
             </div>
             <div>
-              <PhotoList Photos={this.state.photos} handleClick={this.handleClick} />
+              <PhotoList Photos={this.state.photos} handleCarouselClick={this.handleCarouselClick} />
```

Comment:
If you do the destructuring noted in other comment, you can use the variable directly here too :+1: